lib/apptimize_flutter.dart

```diff
@@ -270,16 +334,45 @@ class Apptimize {
   /// the test or instant update. If there are no winners or instant updates,
   /// an empty dictionary is returned. If `startApptimize` has not been called
   /// yet `null` is returned.
-  static Future<Map<String, ApptimizeInstantUpdateOrWinnerInfo>>
+  static Future<Map<String, ApptimizeInstantUpdateOrWinnerInfo?>?>
       get instantUpdateAndWinnerInfo async {
-    final Map<dynamic, dynamic> apptimizeTestInfos =
+    final Map<dynamic, dynamic>? result =
         await _channel.invokeMethod('getInstantUpdateAndWinnerInfo');
+    if (result == null) {
+      return null;
+    }
+
+    final entries = result.entries;
```

Comment:
probably we can get rid of it using result?.entries

Reply:
Null is actually a valid response to the call to `getInstantUpdateAndWinnerInfo`, so we don't want to log a problem in that case. I just want to log something in the case where we got a response but it was not the right response.